pinot-common/src/main/java/org/apache/pinot/common/function/AggregationFunctionType.java

```diff
@@ -48,7 +48,8 @@
   DISTINCTCOUNTRAWHLLMV("distinctCountRawHLLMV"),
   PERCENTILEMV("percentileMV"),
   PERCENTILEESTMV("percentileEstMV"),
-  PERCENTILETDIGESTMV("percentileTDigestMV");
+  PERCENTILETDIGESTMV("percentileTDigestMV"),
+  DISTINCT("distinctValues");
```

Comment:
Wrong merge?

Reply:
yes, removed changes. thanks for catching